extensions/card4l-eo/README.md

```diff
@@ -0,0 +1,184 @@
+- *Optical Surface Reflectance* (SR) products in version 5.0
+  ([PDF](http://ceos.org/ard/files/PFS/SR/v5.0/CARD4L_Product_Family_Specification_Surface_Reflectance-v5.0.pdf),
+  [Word](http://ceos.org/ard/files/PFS/SR/v5.0/CARD4L_Product_Family_Specification_Surface_Reflectance-v5.0.docx))
+  *or*
+- *Optical Surface Temperature* (ST) products in version 5.0
+  ([PDF](http://ceos.org/ard/files/PFS/ST/v5.0/CARD4L_Product_Family_Specification_Surface_Temperature-v5.0.pdf),
+  [Word](http://ceos.org/ard/files/PFS/ST/v5.0/CARD4L_Product_Family_Specification_Surface_Temperature-v5.0.docx)).
+
+**Document structure:** In general, the fields required in this extension are required to either meet the *threshold (minimum) requirements* by the CEOS CARD4L specification *or* are required fields in STAC. Any additional optional field provided may lead to a higher percentage of the CARD4L *target (desired) requirements* met.
+
+The column *Field Name* refers to the STAC field names. The column *Req.* refers to the requirement number in the CARD4L specification.
+
+**STAC Extensions:** This extension makes use of a number of existing STAC extensions:
+
+- [EO](../eo/README.md)
+- [File](../file/README.md)
+- [Processing](../processing/README.md) (optional)
+- [Projection](../projection/README.md)
+- [View](../view/README.md)
+
+**Additional resources:**
+
+- [Examples](examples/) (ToDo)
+- [JSON Schema](json-schema/) (ToDo)
+
+## STAC Items
+
+STAC Items must always be valid, but not all STAC Item requirements are covered here, only additional requirements and mappings to fulfill the CARD4L requirements are listed here:
+
+| Field Name      | Description                                                  | Req.  |
+| --------------- | ------------------------------------------------------------ | ----- |
+| stac_extensions | **REQUIRED.** Must contain all extensions used, for the product at least the following values: `card4l-eo`, `eo`, `projection` and  `view`. You may add `processing` and `file` if used. | *n/a* |
+| geometry        | **REQUIRED.** The geometry of the acquisition.               | 1.4   |
+| bbox            | **REQUIRED.** The bounding box of the acquisition.           | 1.4   |
+
+### STAC Item Properties
+
+#### CARD4L
+
+| Field Name                         | Data Type | Description                                                  | Req.  |
+| ---------------------------------- | --------- | ------------------------------------------------------------ | ----- |
+| card4l:specification               | string    | **REQUIRED.** The CARD4L specification implemented, either `SR` (Optical, Surface Reflectance) or `ST` (Optical, Surface Temperature). | *n/a* |
+| card4l:specification_version       | string    | **REQUIRED.** The CARD4L specification version. Currently always `5.0`. | *n/a* |
+| card4l:northern_geometric_accuracy | number    | An estimate of the northern geometric accuracy in meters.    | 1.8   |
```

Comment:
Previously discussed with @m-mohr , but geometric accuracy I think is more generally applicable outside of CARD4L.  It may make sense to have an extension that includes geometric and radiometric accuracies.  Does it fit in with the `processing` extension?

Reply:
Agreed, but not sure about the new place. It's also included in Card4l Sar